content/_index/embed_subscribe.md

```diff
@@ -0,0 +1,19 @@
++++
+fragment = "embed"
+#disabled = true
+date = "2017-10-07"
+weight = 1050
+background = "light"
+form_name = "subscribe"
+
+#title = ""
+#subtitle  = ""
+
+# Embed a form via an iframe
+# Mailerlite is one example of a working provider.
+# There are others such as convertkit.
+# Only necessity is for them to use iframes.
+media = '<iframe src="https://landing.mailerlite.com/webforms/landing/m0h9d7" style="border: none; width: 100%; height: 178px;"></iframe>'
```

Comment:
This is a "production" form. So not sure, that we should keep it within the starter.
Maybe move the actual form to an example form, if mailerlite provides something like that. or we move it to some random one: `/landing/1234`?